### PR TITLE
Add tinyobjloader.

### PR DESCRIPTION
**Pull request by benjaminrwilson**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by benjaminrwilson** [comment]
@conda-forge/staged-recipes, this is ready for review.

**Comment by benjaminrwilson** [comment]
@chrisburr -- would you mind taking a look at this when you get a chance? Very similar to: https://github.com/conda-forge/tinygltf-feedstock/ in terms of structure.

File: recipes/tinyobjloader/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 19ee82cd201761954dd833de551edb570e33b320d6027e0d91455faf7cd4c341
+
+build:
+  number: 0
+  noarch: generic
```
Comment (chrisburr) [inline recipes/tinyobjloader/meta.yaml:14]: As the install location for Windows and unix is different this one can't be noarch unfortunately:
```suggestion
```

Comment (benjaminrwilson) [reply recipes/tinyobjloader/meta.yaml]: Thanks for taking a look, @chrisburr. Should be good to go now.

**Pull request merged**